apps/studio-client/src/main/joo/com/coremedia/labs/translation/gcc/studio/GccProcessDefinitions_de.properties

```diff
@@ -59,7 +59,7 @@ translationResultXliff_Button_text=ZIP Datei mit Detailinformationen herunterlad
 workflow_contentSetIssue_SUCCESS_sg_text=Das \u00DCbersetzungsergebnis wurde erfolgreich importiert.
 workflow_contentSetIssue_SUCCESS_pl_text=Die \u00DCbersetzungsergebnisse wurden erfolgreich importiert.
 #confirm cancellation dialog
-confirm_cancellation=Wollen Sie wirklich den Workflow "{0}" abbrechen? Dies wird zu einem Abbruch des GlobalLink-Auftrags f\u00FChren.
+confirm_cancellation=Wollen Sie wirklich die ausgew\u00E4hlten Workflows abbrechen? Dies wird zu einem Abbruch des GlobalLink-Auftrags f\u00FChren.
```

Comment:
In contrast to the English message, this does not incorporate possible "single selections". Can we use the `sg_text`, `pl_text` pattern here?